CMakeLists.txt

```diff
@@ -125,7 +130,8 @@ list(FILTER UTILS_HEADER EXCLUDE REGEX cgal*)
 add_library(utils STATIC ${UTILS_SRCS} ${UTILS_HEADER})
 set_property(TARGET utils PROPERTY CXX_STANDARD 14)
 set_property(TARGET utils PROPERTY CXX_STANDARD_REQUIRED ON)
-target_link_libraries(utils igl::core igl::opengl igl::cgal igl::triangle spdlog Qt5::Core Qt5::Widgets spdlog)
+include_directories(external/libigl/external/glfw/include)
```

Comment:
You shouldn't need to set this globally. The target `glfw` should handle setting up include paths. You can remove this line.